boskos/deployment.yaml

```diff
@@ -35,24 +38,24 @@ kind: PersistentVolumeClaim
 metadata:
   labels:
     app: boskos
-  name: boskos-storage
+  name: boskos-volume-boskos-0
```

Comment:
We don't need this anymore because the StatefulSet controller will stamp out a PVC